pkg/operator/controller/ingress/controller_test.go

```diff
@@ -0,0 +1,263 @@
+package ingress
+
+import (
+	"reflect"
+	"testing"
+
+	configv1 "github.com/openshift/api/config/v1"
+	operatorv1 "github.com/openshift/api/operator/v1"
+)
+
+func TestTLSProfileSpecForSecurityProfile(t *testing.T) {
+	invalidTLSVersion := configv1.TLSProtocolVersion("abc")
+	invalidCiphers := []string{"ECDHE-ECDSA-AES256-GCM-SHA384", "invalid cipher"}
+	validCiphers := []string{"ECDHE-ECDSA-AES256-GCM-SHA384"}
+	testCases := []struct {
+		description  string
+		profile      *configv1.TLSSecurityProfile
+		valid        bool
+		expectedSpec *configv1.TLSProfileSpec
+	}{
+
+		{
+			description:  "default (nil)",
+			profile:      nil,
+			valid:        true,
+			expectedSpec: configv1.TLSProfiles["Intermediate"],
```

Comment:
How about a [constant](https://github.com/openshift/api/blob/master/config/v1/types_tlssecurityprofile.go#L127) here and below?

Reply:
Will fix here (and in other files) in the next push.